pages/about.html

```diff
@@ -4,9 +4,20 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <!-- <link href="styles/style.css" rel="stylesheet"> -->
+  <link href="styles/about.css" rel="stylesheet">
+  
   <title>Document</title>
 </head>
+<h1>A little about me</h1>
+<h2>Esther "Hoku" McAngus</h2>
 <body>
+  I am Friendly and Dependable. I Enjoy working closely with others and offering help when needed.
+  I am able to take constructive criticism and use it to the best of my ability.
+  Some activities that I enjoy doing are binge watching episodes on Netflix, spending time with my Ohana and discovering new parks near me.
+
   
 </body>
```

Comment:
This could look like:
```html
<body>
    <h1>A little about me</h1>
    <main>
        <section>
            <h2>Esther "Hoku" McAngus</h2>
            <p>  I am Friendly and Dependable. I Enjoy working closely with others and offering help when needed.
  I am able to take constructive criticism and use it to the best of my ability.
  Some activities that I enjoy doing are binge watching episodes on Netflix, spending time with my Ohana and discovering new parks near me.
            </p>
        </section>
    </main>
</body>